doc/source/whatsnew/v0.21.0.txt

```diff
@@ -830,6 +830,7 @@ Other API Changes
 - Restricted DateOffset keyword arguments.  Previously, ``DateOffset`` subclasses allowed arbitrary keyword arguments which could lead to unexpected behavior.  Now, only valid arguments will be accepted. (:issue:`17176`).
 - Pandas no longer registers matplotlib converters on import. The converters
   will be registered and used when the first plot is draw (:issue:`17710`)
+- ``NaT`` division with :class:`datetime.timedelta` will now return ``NaN`` instead of raising (:issue:`17876`)
```

Comment:
so will move to 0.21.1 when its available

Reply:
can you move to 0.22.0 other api changes

Reply:
Done.  Out of curiosity, why 0.22.0 instead of 0.21.1?